share/instruments/generateCVS.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# If you get Unicode errors on Windows, try setting the environment variable
+# PYTHONIOENCODING=utf-8. More info at https://stackoverflow.com/a/12834315
+
+import xml.etree.ElementTree as ET
+import os
+
+
+# Create instruments.csv (must specify encoding and line ending on Windows)
+f = open('instruments.csv', 'w', newline='\n', encoding='utf-8')
+
```

Comment:
Can you add (something like)  `f.write("SEP=,\n")` here, so locales with `,` as the decimal separator can open the CSV too?